lib/commands/administration.js

```diff
@@ -134,6 +134,99 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('clearUserRoles', { prevSubject: 'optional' }, (...args) => {
+  const $args = normalizedArgumentsWithAuth(args);
+  const [auth, user, clientOptions] = $args;
+
+  const options = { ...clientOptions, ...{ failOnStatusCode: false } };
```

Comment:
@CGuether When configuring `{failOnStatusCode: false}`, tests will not fail if request fails. If disabling `failOnStatusCode`, there should be some error handling for `c8yclient` implemented. This could be for example check for 404 if a user does not exist, etc. Maybe even use of `{failOnStatusCode: false}` is not intended at all. Please check.

Error handling could be implemented using:
`expect(respinse.status).to.be.oneOf([204, 404]);`

Reply:
I have removed `failOnStatusCode` to be set to `false` per default. Instead, user must exist for this command to be executed successfully independent of `failOnStatusCode`.